### ISSUE TITLE
startApp() - subprocess call will never return

### ISSUE THREAD

**Issue by tnguyen-cribl** [abesvt/Test#2]
`subprocess.run()` is a blocking call and will not return until the process is terminated or dies.
Also, there appears to be a sleep of 10,000 seconds after each call.

**Comment by abesvt** [comment]
Yeah, this is the part that I spent maximum time on. I was trying out different methods like subprocess.popen, os.system etc with timeout and such to get it to return back after running the commands and couldnt get it to work. 
I added the delay so that it can atleast return back after that but that didnt work either.

**Comment by tnguyen-cribl** [comment]
Yeah, `subprocess` methods are usually synchronous, so you'll need something to execute the apps async and then terminate on teardown as the servers will not kill themselves.
